jest.config.js

```diff
@@ -2,13 +2,14 @@ module.exports = {
   automock: false,
   clearMocks: true,
   errorOnDeprecated: false,
-  moduleFileExtensions: ["js", "json"],
+  moduleFileExtensions: ["ts", "tsx", "js", "json"],
   moduleNameMapper: {
     "^.+\\.css$": "identity-obj-proxy",
   },
   testEnvironment: "jsdom",
   timers: "modern",
+  // might also have to change testMatch
   transform: {
-    "^.+\\.(js|jsx)$": ["babel-jest", { configFile: "./babel.jest.config.js" }],
+    "^.+\\.(js|jsx|ts|tsx)$": ["babel-jest", { configFile: "./babel.jest.config.js" }],
```

Comment:
Similarly, this tells Jest to _also_ transform (transpile) `.ts` and `.tsx` files with Babel.